internal/clients/connect/capabilities.go

```diff
@@ -44,7 +44,13 @@ func checkRequirementsFile(base util.AbsolutePath, requirementsFilename string)
 	return nil
 }
 
-func (c *ConnectClient) CheckCapabilities(base util.AbsolutePath, cfg *config.Config, log logging.Logger) error {
+func (c *ConnectClient) CheckCapabilities(base util.AbsolutePath, cfg *config.Config, contentID *types.ContentID, log logging.Logger) error {
+	if contentID != nil && *contentID != "" {
+		err := c.ValidateDeploymentTarget(*contentID, log)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
If we are updating an existing deployment, we'll run those validation tests (now located within the ConnectClient package)